classes/Adapters/MySQL/ActiveRecordAdapter.php

```diff
@@ -595,4 +622,23 @@ public function GetReportGrouped($_siteId, $_startTimestamp, $_endTimestamp, $_a
 
         return $grouped_types;
     }
+
+    /**
+     * TRUNCATE temp table `tmp_users` and populate with users
+     * It is used in the query of the above function
+     */
+    private function TempUsers($tableUsers)
+    {
+        $_wpdb = $this->connection;
+        $sql = "TRUNCATE $tableUsers";
```

Comment:
@carmelopreeostudios we can't use `TRUNCATE` - this requires the `DROP` function - use `DELETE FROM $tableUsers` instead - since we don't have Primary Key it IDs will always be the same